testing/xfpga/test_umsg_c.cpp

```diff
@@ -132,6 +164,14 @@ class umsg_c_p
 
   virtual void TearDown() override {
     EXPECT_EQ(fpgaDestroyProperties(&filter_), FPGA_OK);
+
```

Comment:
handle_ is being opened in the SetUp, but isn't being closed here in TearDown.  Additionally, in some tests (test_umsg_drv_07), handle_ is open'ed again.  I think we have some resource leaks and potential memory errors that need to be addressed.